src/tuner/nccl_ofi_tuner.c

```diff
@@ -428,15 +428,15 @@ ncclResult_t nccl_ofi_tuner_get_coll_info(void *context,
 
 		in_out = is_inside_region(p, &nccl_ofi_tuner_ctx->regions[collType][i]);
 		if (in_out >= 0) {
-			table[algorithm][protocol] = 0.0;
+			table[algorithm][protocol] = 0.0f;
 
 			NCCL_OFI_INFO(NCCL_TUNING,
-				      "Choosing algo %d proto %d with cost %.8f µsecs for coll %d size %ld.",
-				      algorithm,
-				      protocol,
-				      table[algorithm][protocol],
-				      collType,
-				      nBytes);
+			              "Choosing algo %d proto %d with cost %.8lf µsecs for coll %d size %ld.",
```

Comment:
`f` and `lf` seem to be identical... not sure why this needs to be changed.

Reply:
https://stackoverflow.com/a/210591